pkg/model/manifest.go

```diff
@@ -124,6 +124,9 @@ var (
 		"helm/charts",
 	}
 	manifestSubPath = []string{
+		"manifest",
+		"manifest.yml",
+		"manifest.yaml",
```

Comment:
As the files are processed in the order defined here and in order to keep current preference of files, can you include them at the end of the list?

Reply:
Sure, I have made the changes!

Reply:
Thanks for reviewing @ifbyol!